src/iterators/mod.rs

```diff
@@ -65,34 +88,35 @@ pub struct IterationValue<T: Counter> {
 
 impl<T: Counter> IterationValue<T> {
     /// Create a new IterationValue.
-    pub fn new(value: u64, quantile: f64, quantile_iterated_to: f64, count_at_value: T,
+    pub fn new(value_iterated_to: u64, quantile: f64, quantile_iterated_to: f64, count_at_value: T,
                count_since_last_iteration: u64) -> IterationValue<T> {
         IterationValue {
-            value,
+            value_iterated_to,
             quantile,
             quantile_iterated_to,
             count_at_value,
             count_since_last_iteration
         }
     }
 
-    /// The lowest value stored in the current histogram bin
-    pub fn value(&self) -> u64 {
-        self.value
+    /// The value iterated to. Some iterators provide a specific value inside the bucket, while
+    /// others just use the highest value in the bucket.
+    pub fn value_iterated_to(&self) -> u64 {
```

Comment:
Maybe this rename-for-clarity is not worth the compatibility concern?

Reply:
I think it's fine.